crates/web5/src/jose.rs

```diff
@@ -62,6 +62,7 @@ pub struct JwtClaims {
 impl FromJson for JwtClaims {}
 impl ToJson for JwtClaims {}
 
+#[allow(dead_code)]
```

Comment:
was linting or something throwing an error here?

Reply:
yup, should we change the lint config to not throw on this stuff?

Reply:
I'm confused why it would be throwing this tho, because we use this in VC JWTs and VP JWTs

Reply:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/19c86c90-2e7d-4c32-b638-9df87d224a0d">